bundles/org.eclipse.swt/Readme.macOS.md

```diff
@@ -82,13 +82,30 @@ More information on how to use the generator is available [here](https://www.ecl
 this information into this readme.
 
 
-Compiling native libs
----------------------
+## Building Natives
 
-This is done from within the binaries project.
+### Building 64bit natives for Cocoa
 
-    cd <eclipse.platform.swt>/binaries/org.eclipse.swt.cocoa.macosx.x86_64
-    mvn clean package -Dnative=cocoa.macosx.x86_64
+You need to install the following on your system:
 
+* TBD
+* Java 17 JDK
 
-Happy development!
\ No newline at end of file
+### Building and Testing locally
+
+Import and build the SWT project `bundles/org.eclipse.swt` and `binaries/org.eclipse.swt.cocoa.macosx.${arch}` in Eclipse.
+
+See **Building and Testing locally** in [Readme.md](Readme.md)
+
+Alternatively run the `build.sh` script under `org.eclipse.swt\Eclipse SWT PI\cocoa\library\` directory.
+To use the locally build natives run the script with the 'install' and `clean` arguments (`./build.sh clean install`).
```

Comment:
Does not work, see explanation for Linux.

Reply:
Removed in d556545966b1beefd598027b20675dfa34cde7e6